.ddev/config.yaml

```diff
@@ -0,0 +1,256 @@
+name: ddev.com
```

Comment:
It's important not to use a real on-internet name in these situations. It causes enormous grief and confusion.

However, I usually recommend leaving out the `name` when providing a pre-populated config.yaml so it can just use the project name of the enclosing directory.

```suggestion
```